libparsec/crates/testbed/src/template/events.rs

```diff
@@ -1759,10 +2173,14 @@ impl TestbedEventCreateOrUpdateFolderManifestVlob {
         if builder.check_consistency {
             utils::assert_organization_bootstrapped(&builder.events);
             utils::assert_device_exists_and_not_revoked(&builder.events, &author);
-            utils::assert_realm_exists_and_not_under_reencryption(&builder.events, realm);
+            utils::assert_realm_exists(&builder.events, realm);
             utils::assert_realm_member_has_write_access(&builder.events, realm, author.user_id());
         }
 
+        let (key_index, key) = utils::realm_keys(&builder.events, realm).last().expect(
+            "Realm must have had at least one key rotation vlob create/update is possible !",
```

Comment:
```suggestion
            "Realm must have at least one key rotation for vlob create/update to be possible !",
```

Reply:
`have had` is past tense, I've corrected the rest